adl

```diff
@@ -230,8 +246,7 @@ animedl() { #{{{
     choice_arg()   { [[ -z "$yes" ]] && echo "" || echo "-c 1" ;}
     player_arg="$1"
     down_arg()     { [[ "$download" == "0" ]] && echo "--play $player_arg"   || echo "" ;}
-    provider_arg() { [[ "$provider" != ""  ]] && echo "--provider $provider" || echo "" ;}
-
+    provider_arg() { [[ "$provider" != ""  ]] && echo "--provider $provider" || echo "--provider animerush" ;} # setting default provider to animerush because it gives far better results
```

Comment:
I would most likely revert this, as it has been covered by readme. Another option is to just set this at the [top](https://github.com/RaitaroH/adl/blob/master/adl#L30) of the script.

Reply:
Yeah you can do that ,we can let users decide which provider they wanna use